### PR TITLE
Add test case for UserStatusService

### LINKED ISSUE #12556: Add test case for UserStatusService

### PR DESCRIPTION
**Pull request by zhujunxxxxx**
Fixes #12556.

Changes proposed in this pull request:
- Add test case for UserStatusService

### CODE REVIEW

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/registry/status/user/service/UserStatusServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UserStatusServiceTest {
```
Comment (tristaZero) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/registry/status/user/service/UserStatusServiceTest.java:34]: Hi @zhujunxxxxx I guess final class here is better.

**Review by tristaZero — CHANGES_REQUESTED** [review]

**Comment by tristaZero** [comment]
Hi @zhujunxxxxx I saw you're dealing with ShardingSphere repo, nice work and welcome!

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/registry/status/user/service/UserStatusServiceTest.java (hunk 1 above)
Comment (zhujunxxxxx) [reply shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/registry/status/user/service/UserStatusServiceTest.java]: OK I will fix it and I'm back ^_^

**Review by tristaZero — APPROVED** [review]

File: shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/registry/status/user/service/UserStatusServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    @Test
+    public void assertPersist() {
+        Collection<ShardingSphereUser> users = new LinkedList<>();
+        users.add(new ShardingSphereUser("test", "", "%"));
+        UserStatusService userStatusService = new UserStatusService(repository);
+        userStatusService.persist(users);
+        verify(repository).persist(anyString(), anyString());
+    }
+
```
Comment (tristaZero) [inline shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/registry/status/user/service/UserStatusServiceTest.java:47]: Hi @zhujunxxxxx Generally, it's fine to merge it. But I suggest removing this unexpected blank line.

**Pull request merged**